src/renderer/components/SettingsPage.tsx

```diff
@@ -1228,35 +1230,36 @@ class SettingsPage extends React.PureComponent<Props, State> {
         }
 
         return (
-            <div
-                className='container-fluid'
-                style={{
-                    height: '100%',
-                }}
+            <Modal
```

Comment:
dont we use Modals from mattermost/components here as well?

Reply:
Not at this time no, but I would absolutely like to. I was planning on looking into those when we redesign the modal to fit @abhijit-singh's new designs.